src/pacmod3_core.cpp

```diff
@@ -29,6 +29,11 @@ const uint32_t AS::Drivers::PACMod3::TurnSignalCmdMsg::CAN_ID = 0x130;
 const uint32_t AS::Drivers::PACMod3::WiperCmdMsg::CAN_ID = 0x134;
 const uint32_t AS::Drivers::PACMod3::RearPassDoorCmdMsg::CAN_ID = 0x140;
 
+constexpr uint8_t AS::Drivers::PACMod3::SystemCmdBool::DATA_LENGTH;
```

Comment:
Why do you need to declare this again here if it was declared in pacmod3_core as `static`?

Reply:
`static` data members that are declared in a header file (regardless of whether they have a default value or not) have to have an implementation in the `cpp` file or the linker will complain. Here's a more detailed explanation: https://softwareengineering.stackexchange.com/questions/145299/why-the-static-data-members-have-to-be-defined-outside-the-class-separately-in-c